app/views/homepage/index.html.erb

```diff
@@ -103,18 +102,20 @@
             } %>
           </a>
           <h3 class="home-promo__heading">
-            <a href="/coronavirus" class="govuk-link home-promo__link">
-              <%= t('homepage.index.coronavirus') %>
+            <a href="https://www.gov.uk/government/news/celebrating-pride-month-june-2021" class="govuk-link home-promo__link">
+              <%= t('homepage.index.pride_month') %>
             </a>
           </h3>
           <p class="home-promo__text">
-            <%= t('homepage.index.coronavirus_restrictions') %>
+            <%= t('homepage.index.pride_month_description') %>
           </p>
         </div>
-
+      </div>
+      
+      <div class="govuk-grid-row">
```

Comment:
All the `<div class="govuk-grid-column-one-third">` columns should be contained inside this `govuk-grid-row` wrapper or layout issues will appear